test/test_pruning_op.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import hypothesis.strategies as st
+from hypothesis import given
+import numpy as np
+import torch
+from torch.testing._internal.common_utils import TestCase
+
+class PruningOpTest(TestCase):
+
+    def _test_rowwise_prune_op(self, embedding_rows, embedding_dims, indices_type):
```

Comment:
Please add a test to check if this op is scriptable. Since it takes a dtype as an argument, I am not sure if it is

Reply:
I think this comment doesn't apply as it's just a pytorch c++ op and not a nn.Module.